tests/unit_tests/test_plots.py

```diff
@@ -12,6 +12,7 @@ def myplot():
     plot.filename = 'myplot'
     plot.type = 'slice'
     plot.basis = 'yz'
+    plot.outline = True
```

Comment:
This is just setting an attribute on the `Plot` class that doesn't do anything. If you want to test it, you can modify `test_plot` in unit_tests/test_universe.py

Reply:
sorry got mixed up in the plot, plotter, universe.plot world. Fixed now